paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetActivity.kt

```diff
@@ -86,11 +90,28 @@ internal class CustomerSheetActivity : AppCompatActivity() {
             }
         }
 
+        lifecycleScope.launch {
+            viewModel.action.collect {
+                handleAction(it)
+            }
+        }
+
         onBackPressedDispatcher.addCallback {
             finishWithResult(InternalCustomerSheetResult.Canceled)
         }
     }
 
+    private fun handleAction(action: CustomerSheetAction?) {
```

Comment:
We should try to model this as a non-nullable action. Maybe we can use a `SharedFlow` in the view model or model the action as part of the state?